src/components/Home.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+//import './App.css';
+//import './index.css';
+
+function Home() {
+  return (
+      <div className='Homepage overflow-hidden justify-center'>
+        
+        <center>
+          <div className='bg-blue w-100 h-[690px] grid-cols-1 z-10 -mt-72  items-center border-b-[22px] border-orange overflow-x-hidden'>
+            <div className='z-30 relative pt-72 text-center font-jomhuria tracking-wide text-[8rem]'>
+              <h1 className='text-white'>FOOD FROM TV.</h1>
+              <h1 className='text-orange -mt-28'>ALL IN ONE PLACE.</h1>
+            </div>
+              <div className='-mt-6 relative w-1/4'>
+                <form>   
+                    <label htmlFor="default-search" className="mb-2 text-sm font-medium text-white sr-only">Search</label>
+                    <div className="relative">
+                        <div className="absolute inset-y-0 start-0 flex items-center ps-3 pointer-events-none">
+                            <svg className="w-4 h-4 text-gray-500 dark:text-gray-400" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 20 20">

```

Comment:
So one thing I noticed about the logo SVGs is that they are not actually vector graphics, they are very large color matrices (4.8MB). We want to avoid these, but nbd since the logo is temporary.